index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>A Product Landing Page</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+    <header id="header">
+    <img id="header-img" src="eye.png" alt="">
+    <nav id="nav-bar">
+    <a class="nav-link" href="#mail">Home</a>
+    <a class="nav-link" href="#text-container">About</a>
+    <a class="nav-link" href="#video-section">Media</a>
+    </nav>
+    </header>
```

Comment:
pls kindly add the hover and active  effect on the links in other  to improve the user experience when on the page

Reply:
I think you should add hover and active pseudo classes on the links so as to improve the user experience when on the page.